cypress/support/commands.js

```diff
@@ -23,3 +23,12 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+Cypress.Commands.add('findByPlaceholder', (placeholder) => {
+    cy.get(`[placeholder="${placeholder}"]`);
+  });
```

Comment:
Consider removing the extra whitespace before the closing bracket of the 'findByPlaceholder' command for better readability 🧹.